duneana/CAFMaker/CAFMaker.fcl

```diff
@@ -44,10 +44,13 @@ dunefd_cafmaker:
     GenieGenModuleLabel:     "generator"
     POTModuleLabel:          "generator"
     FlashModuleLabel:        "opflash"
+    DirectionRecoLabelNue:   ""
+    DirectionRecoLabelNumu:  ""
     CalorimetryAlg:          @local::dune10kt_calorimetryalgmc
     FidVolCut:	             3.0
     NuECut:                  0.8
     NuMuCut:                 0.8
+    PandoraNuVertexModuleLabel: "pandora"
     generated_systematic_provider_configuration: @local::generated_systematic_provider_configuration
 }
 
```

Comment:
Just a general comment here, I _think_ some of these names are different in the new production? like cvneval -> cvneva? So maybe check them. It might just be me using the wrong samweb def though...